hack/make-rules/helpers/go2make/BUILD

```diff
@@ -0,0 +1,41 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "go2make.go",
+        "pkgwalk.go",
+    ],
+    importpath = "k8s.io/kubernetes/hack/make-rules/helpers/go2make",
+    visibility = ["//visibility:private"],
+    deps = ["//vendor/github.com/spf13/pflag:go_default_library"],
+)
+
+go_binary(
+    name = "go2make",
+    embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = ["pkgwalk_test.go"],
```

Comment:
Your test is failing because it is trying to open a ./testdata/dir-without-gofiles which doesn't exist.

Can you try adding `data = [":package-srcs"]` here? You can also try the `data = glob(["testdata/**"])`, but that's included in package-srcs.

Although I suspect it might make the dir hack/make-rules/helpers/go2make/testdata